sw/device/silicon_creator/rom_ext/e2e/ownership/defs.bzl

```diff
@@ -0,0 +1,46 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//rules/opentitan:defs.bzl",
+    "opentitan_test",
+)
+
+def ownership_transfer_test(
+        name,
+        srcs = ["//sw/device/silicon_creator/rom_ext/e2e/verified_boot:boot_test"],
+        exec_env = {
+            "//hw/top_earlgrey:fpga_cw310_rom_ext": None,
```

Comment:
nit: It would also be nice to also support the cw340 target.

`//hw/top_earlgrey:fpga_cw340_rom_ext`

Reply:
Created #24504.